content/docs/reference/spec/migration/platform-api-0.5-0.6.md

```diff
@@ -0,0 +1,23 @@
++++
+title="Platform API 0.5 -> 0.6"
++++
+
+<!--more-->
+
+This guide is most relevant to platform operators.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/platform%2Fv0.6) for platform API 0.6 for the full list of changes and further details.
```

Comment:
We need to make sure that the spec release link is working before we merge this PR.